File: TChores/app/src/main/java/me/tseng/studios/tchores/java/viewmodel/MainActivityViewModel.java
package me.tseng.studios.tchores.java.viewmodel;

import android.arch.lifecycle.ViewModel;

import me.tseng.studios.tchores.java.Filters;


public class MainActivityViewModel extends ViewModel {

    private boolean mIsSigningIn;

    public MainActivityViewModel() {
        mIsSigningIn = false;

    }

    public MainActivityViewModel(String currentUserName) {
        mIsSigningIn = false;
    }

    public boolean getIsSigningIn() {
        return mIsSigningIn;
    }

    public void setIsSigningIn(boolean mIsSigningIn) {
        this.mIsSigningIn = mIsSigningIn;
    }


}

File: TChores/app/src/main/java/me/tseng/studios/tchores/kotlin/viewmodel/MainActivityViewModel.kt
package me.tseng.studios.tchores.kotlin.viewmodel

import android.arch.lifecycle.ViewModel
import me.tseng.studios.tchores.kotlin.Filters

/**
 * ViewModel for [me.tseng.studios.tchores.MainActivity].
 */

class MainActivityViewModel : ViewModel() {

    var isSigningIn: Boolean = false
    var filters: Filters = Filters.default
}
